rpms/s3testrpm/buildrpm.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh
```

Comment:
why not use python script instead of shell ?

Reply:
So that we are consistent with rest of the rpms / framework
Its anyway not part of mini provisioner.